tests/functional/pages/surveys/question.page.js

```diff
@@ -33,6 +33,11 @@ class QuestionPage {
     return browser.click('a[id="top-previous"]')
   }
 
+  next() {
```

Comment:
Could you load a shorter survey? seems like a lot of overhead just to get to the thank-you page.